### DISCUSSION TITLE
Possible to use Conbee II as router?

### DISCUSSION THREAD

**Discussion by JensBee** [Koenkk/zigbee2mqtt#12685] [Devices]
Hi,

is it possible to use a conbee 2 USB stick as router?

I configuring it (via deconz) on a different machine as router and tried to join the network. The device gets detected but unfortunately flagged as unsupported. Loading the map it gets marked as router and connected.

Just to be sure I'm not doing something wrong.. is this setup supported or do I need to add support for this device?

Best, Jens

**Comment by JensBee** [comment]
Connecting the device fails with `node descriptor request failed`. So I don't know if it's unsupported, since it won't connect.

I tried different firmware Versions, with cable, near/far .. without luck. Maybe someone else has a hint ;)

**Reply by lux73** [reply]
if there is any Router Firmware available?

if not then it's not possible to Act as Router.

Any other Stick's e.g. CC251 / CC2652x there is dedicated Router Firmware available - these must be flashed before using it as Router

**Reply by JensBee** [reply]
The vendor application allows to configure the device as router or coordinator. A firmware flash should not be possible.
My impression is that the router mode is not well supported (by vender not z2m).

**Comment by akshaydogra1** [comment]
Even i am looking for a firmware for Conbee II which will convert it from coordinator to router. Unless the vendor releases a firmware, i do not think that it is possible.

**Comment by rainbowbreeze** [comment]
I'm also looking for a solution - any news?
